### PR TITLE
Added DropdownButton to the widget catalog

### PR DESCRIPTION
**Pull request by awazgyawali**
Addresses this issue https://github.com/flutter/flutter/issues/14460
We might need to add "Spinner" word somewhere to make Android developers easy to find the widget.

### CODE REVIEW

**Comment by awazgyawali** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

I signed it!

File: src/_data/catalog/widgets.json [hunk 1]
```diff
@@ -1608,5 +1608,17 @@
     ],
     "link": "https://docs.flutter.io/flutter/widgets/AnimatedWidgetBaseState-class.html",
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "DropdownButton",
+    "description": "A input field to choose an item from a list.",
```
Comment (Sfshaza) [inline src/_data/catalog/widgets.json]: "A input field to choose..." => "An input field for choosing..."

I thought the idea was to add Spinner to this description for Android users?

File: src/_data/catalog/widgets.json [hunk 2]
```diff
@@ -1608,5 +1608,17 @@
...
+    "categories": [
+      "Material Components"
+    ],
+    "subcategories": [
+      "Input and selections"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/DropdownButton-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
```
Comment (Sfshaza) [inline src/_data/catalog/widgets.json]: Why a placeholder image?

**Comment by chalin** [comment]
Given that this is not a P0, @awazgyawali could you close this PR and reopen one over the `dash` branch?

File: src/_data/catalog/widgets.json (hunk 2 above)
Comment (sfshaza2) [reply src/_data/catalog/widgets.json]: Nevermind, we use lots of them.

**Comment by awazgyawali** [comment]
OK @chalin

**Comment by MarkOSullivan94** [comment]
Has this been merged in?

**Comment by chalin** [comment]
No, but #1903 was merged and it added `DropdownButton`.

**Pull request closed without merging**